website/views/pages/customers/new-license.ejs

```diff
@@ -29,7 +29,7 @@
                 <strong>Order total</strong>
               </div>
               <div class="ml-auto">
-                <strong>${{(!showQuotedPrice || _.isNaN(formData.numberOfHosts * 12)) ? quotedPrice : formData.numberOfHosts * 12}}.00 /year</strong>
+                <strong>${{(!showQuotedPrice || _.isNaN(formData.numberOfHosts * 24)) ? quotedPrice : formData.numberOfHosts * 24}}.00 /year</strong>
```

Comment:
~~I'd suggest we spell out `* 2.00 * 12` here in both spots  (will make it easier to grep for `2.00` later if this changes again)~~

eh, nevermind, doesn't matter (there are other spots we'd have to find anyway)